### ISSUE TITLE
agegroup for every age

### ISSUE THREAD

**Issue by inbaz** [inbaz/ers#612]
At the moment we're using null as agegroup when the participant is an adult. Since agegroup was added as field to item it would be nice to have an agegroup set for every item to be able to correct the items with agegroup IS NULL.

**Comment by jolieno** [comment]
Can't we close this issue? Since at EJC 2016 we don't use the agegroup tab?

**Comment by inbaz** [comment]
No, please don't. There will be an EJC after Almere. :) Maybe they want to use it.
